bench/formatters.mjs

```
@@ -0,0 +1,30 @@
+ * Formats benchmark results as CSV.
+ * @param {Array<Object>} results
+ */
+export function formatCSV(results) {
+  if (results.length === 0) return;
+  const headers = Object.keys(results[0]).join(",");
+  const rows = results.map((r) =>
+    Object.values(r)
+      .map((v) => (String(v).includes(",") ? `"${v}"` : v))
+      .join(",")
+  );
+  console.log([headers, ...rows].join("\n"));
```

ESLint

[error] 'console' is not defined.

(no-undef)